adhocteam/Head-Start-TTADP
# [TTAHUB-244] Add new page to view approved ARs

## Description of change
Created a new page which shows approved reports. From the /activity-reports landing page, clicking either the report ID or selecting View from the context menu takes you to this new design.

## How to test

- Navigate to /activity-reports. Note that there are new context menu items and that approved reports have new links.
- Navigate to an approved report at /activity-reports/view/${reportId}.
- There are two buttons at the top. Confirm that they work as expected. Confirm all the information presented appears correct and consistent. 
- Check the print view and confirm the same.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-244

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
